pkgs/applications/audio/tuner/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, meson
+, ninja
+, vala
+, glib
+, itstool
+, wrapGAppsHook
+, desktop-file-utils
+, libsoup
+, json-glib
+, geoclue2
+, geocode-glib
+, libgee
+, gtk3
+, pantheon
+, gst_all_1
+, gst_plugins ? with gst_all_1; [ gst-plugins-good gst-plugins-ugly ]
```

Comment:
```suggestion
```

Reply:
Thanks, I followed `rhythmbox`. I'll include your suggestion, and re-push it shortly.